nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/AbstractTestBridgeCommentManager.java

```diff
@@ -91,12 +90,14 @@ protected DocumentModel getCommentedDocument() {
         return BridgeCommentManager.class;
     }
 
+    protected abstract BridgeCommentManager getBridgeCommentManager();
```

Comment:
Why moving it? I found it more useful at the end of class where it is visible than in the middle of it.